packaging/nmstate.spec

```diff
@@ -84,6 +85,7 @@ This package contains the nmstate plugin for OVS database manipulation.
 
 %prep
 %setup -q
+%cargo_prep -V 1
```

Comment:
is this the correct syntax for preparing the project for building  ?https://tmz.fedorapeople.org/guidelines/packaging-guidelines/Rust/#_others

Reply:
This is for RHEL 8 where they do not have Fedora-like fancy macros.

Reply:
Oh. `-V` is for vendoring in RHEL.